src/components/cafes/[id]/RoastingBar/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { roastingBar, roastingSegment, roastingStatus } from './RoastingBar.css';
+
+export const RoastingBar = ({ activeLevel }: { activeLevel: 'light' | 'medium' | 'dark' }) => (
+  <>
+    <div className={roastingBar}>
+      <div
+        className={roastingSegment({
+          roastedLevel: activeLevel === 'light' ? 'light' : undefined,
```

Comment:
```suggestion
          isActive: activeLevel === 'light',
```

이런 식으로 표현하면 가독성이 좋아질 것 같은데 어떤가요?